### PR TITLE
Shorten some lines in Go discovery fragments

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

**Comment by saicheems** [comment]
@broady

File: src/main/resources/com/google/api/codegen/go/discovery_fragment.snip [hunk 1]
```diff
@@ -69,7 +70,8 @@ func main() {
         _ = err
       }
     @else
-      if err := {@methodPath(method)}({@callArg(method)}).Context(ctx).Do(); err != nil {
+      err := {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
+      if err != nil {
         // {@TODO()} Handle error.
         _ = err
```
Comment (broady) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: While you're here, you can remove this line.

**Comment by broady** [comment]
UGH, github. What a horrible experience, reviewing this change.

LGTM.

File: src/main/resources/com/google/api/codegen/go/discovery_fragment.snip (hunk 1 above)
Comment (saicheems) [inline src/main/resources/com/google/api/codegen/go/discovery_fragment.snip]: Done, for all `_ = err`

**Pull request merged**